controllers.js

```diff
@@ -38,21 +37,30 @@ exports.getArticlesById = (request, response, next) => {
 };
 
 exports.getArticleComments = (request, response, next) => {
-  const {article_id} = request.params
+  const { article_id } = request.params;
   fetchArticleById(article_id)
-      .then(() => { 
-          return fetchArticleComments(article_id) 
-      })
-      .then((comments) => {
-      response.status(200).send({comments: comments})
-  }).catch(next)
-}
+    .then(() => {
+      return fetchArticleComments(article_id);
+    })
+    .then((comments) => {
+      response.status(200).send({ comments: comments });
```

Comment:
`send({comments})` will have the same effect here, because the key you want to use and variable name are the same ... syntactic sugar!